framework/graphics/dx12_gpu_va_map.cpp

```diff
@@ -61,7 +62,7 @@ void Dx12GpuVaMap::Remove(format::HandleId resource_id, uint64_t old_start_addre
     }
 }
 
-uint64_t Dx12GpuVaMap::Map(uint64_t address, format::HandleId* resource_id, bool* found) const
+uint64_t Dx12GpuVaMap::Map(uint64_t address, format::HandleId* resource_id, bool* found, uint64_t old_end_address) const
```

Comment:
Could the new argument to `Map` and `FindMatch` be renamed to `minimum_old_end_address` since an exact end address match is not required? `FindMatch` checks `old_end_address <= info.old_end_address`.